### ISSUE TITLE
Exit with error code on any script  errors

### ISSUE THREAD

**Issue by AdamNowotny** [jasmine-contrib/grunt-jasmine-runner#20]
At the moment the task fails only if Jasmine tests failed, but reports success if all tests passed but some JS errors were present. It would be great to have an option to fail the build on any script errors as well.
This should help with catching errors not yet covered by jasmine tests.

Example:

```
Running "jasmine" task
Testing jasmine specs via phantom
Error: Script error
http://requirejs.org/docs/errors.html#scripterror
Error: Script error
http://requirejs.org/docs/errors.html#scripterror
31 specs, 0 failures in 0.305s.

Done, without errors.

ERRORLEVEL=0
```

**Comment by jsoverson** [comment]
Good idea, thanks. This probably won't get implemented in jasmine-runner, as that is being deprecated upon grunt 0.4.0, feel free to open up a new issue https://github.com/gruntjs/grunt-contrib-jasmine
